qt/src/DisplayerToolSelect.cc

```diff
@@ -266,7 +268,7 @@ void NumberedDisplayerSelection::paint(QPainter *painter, const QStyleOptionGrap
 	painter->setRenderHint(QPainter::Antialiasing, false);
 	QRectF r = rect();
 	qreal w = 20. / m_tool->getDisplayer()->getCurrentScale();
-	w = qMin(w, qMin(r.width(), r.height()));
+	w = std::min(w, std::min(r.width(), r.height()));
```

Comment:
Is there actually a rationale for preferring std::{min,max} instead of qMin qMax or is it just personal preference?

Reply:
Qt developers recommend use STL stuff instead of Qt, because we are writing on C++, STL becomes more and more powerful, std::min/max is default "min/max" for C++ developers. There are no reasons to use Qt's min/max.